README.md

```diff
@@ -81,6 +81,44 @@ end
 
 Can be `http` or `https` protocol
 
+## Add your own service
+
+You can make available your own services:
+
+`my_awesome_service.rb`:
+
+```ruby
+module MarkdownVideos::Services
+
+  class MyAwsomeService < ServiceBase
+
+    def self.regexp
+      /(https?:\/\/myaweso.me\/video\/([\w\-]{4,16})\??([\w\-\=]+)?)/
+    end
+
+    def width
+      560
+    end
+
+    def height
+      315
+    end
+
+    def url
+      "https://player.myaweso.me/video/#{id}"
```

Comment:
It make sense because any embed service rendered will probably have a unique ID. It's a convenient way to avoid re-witting that for every single service.